doc/coog_admin_tuto.rst

```diff
@@ -0,0 +1,716 @@
+===============
+Coog Deployment
+===============
+
+**coog-admin** is a utility which allows handling **Coog** deployments and easing its administration. In general, each Linux user gets a **Coog** deployment. All **Coog** deployment related data is stored in two directories:
+
+* **~/coog-admin**: contains source files and scripts which allow launching, updating and watching docker containers. This directory's source files should never be changed.
+
+* **~/coog-data**: contains deployment overloads (specific configuration)  and all data volumes linked to active containers.
+
+In addition to these two directories:
+
+* **~/coog-log**: contains functional logs (batch execution reports, backup procedure reports)
+
+Prerequired setup
+-----------------
+
+- a server running on Linux
+- docker
+- github
+- an email utility
+
+Create user and load source files
+---------------------------------
+
+**Coog** deployment is done via **docker**. 
+
+It is highly advised to run **Coog** in another user than root. First of all, install a user with no administration privileges on the machine. Let that user be *coog-user*.
+
+.. code-block:: python
+
+   sudo adduser coog-user
+
+Add that *coog-user* in **docker** groups
+
+.. code-block:: python
+
+    sudo usermod -aG docker coog-user
+
+Configure **git** for *coog-user*
+
+.. code-block:: python
+
+    su - coog-user
+
+You can either generate an SSH key and link it to a github account, or:
```

Comment:
explain that this will be necessary to build image, otherwise we should make docker login and ask for access to pull coog images